api-gateway.json

```diff
@@ -0,0 +1,134 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Overdose Prevention Society API",
+    "version": "1.0.0",
+    "title": "OPS API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "email": "pritpal@codethechange.ca"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "overdose-prevention-inst.c3hn6roqwxvf.ca-central-1.rds.amazonaws.com",
+  "basePath": "/",
+  "tags": [
+    {
+      "name": "city"
+    },
+    {
+      "name": "site"
+    },
+  ],
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/city": {
+      "get": {
+        "tags": [
+          "city"
+        ],
+        "summary": "Get all cities",
+        "operationId": "city-get-all",
+        "responses": {
+          "200": {
+            "description": "All cities",
+            "schema": {
+              "$ref": "#/definitions/City"
+            }
+          }
+        }
+      }
+    },
+    "/city/{cid}": {
```

Comment:
What's the use case for getting a specific city? I'm thinking that the last endpoint already handles what's happening in this one.